readme.md

```diff
@@ -25,7 +30,7 @@ Your Answer:
 Describe at a high level how we use jQuery to submit a form via AJAX.
 
 Your Answer:
-```text
+```When we use jQuery to submit a form via AJAX it has some advantages. Such as we can process the form without having to refresh the page. It is very useful in single paged apps. We can run both Client and Server validation. Show errors if there are any. Also showing success if a form is submitted correctly. All from one page with out reload.
```

Comment:
You answered the why, but we're looking for the *how*. Check the solution branch for a model answer.